io.sourceforge.trigger_rally.TriggerRally.yaml

```diff
@@ -0,0 +1,65 @@
+app-id: io.sourceforge.trigger_rally.TriggerRally
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+command: trigger-rally
+rename-icon: trigger-rally
+rename-desktop-file: trigger-rally.desktop
+rename-appdata-file: trigger-rally.appdata.xml
```

Comment:
there is no reason to have these rename if the files are in the flathub repository. I think I have mentionned this a few times already.

Reply:
They are already commited using this name. https://sourceforge.net/p/trigger-rally/code/HEAD/tree/data/icon/